src/bci_build/package/appcontainers.py

```diff
@@ -271,6 +271,101 @@
 ]
 
 
+_POSTFIX_FILES = {}
+for filename in (
+    "entrypoint.sh",
+    "entrypoint.sles.sh",
+    "postfix-manpages.md",
+    "smtpd_sender_login_maps",
+    "virtual_alias_domains",
+    "virtual_alias_maps",
+    "virtual_gid_maps",
+    "virtual_mailbox_maps",
+    "virtual_uid_maps",
+    "README.md",
+    "README.sles.md",
+):
+    if filename.startswith("README."):
+        if os_version == OsVersion.TUMBLEWEED and filename == "README.md":
+            _POSTFIX_FILES[filename] = (
+                Path(__file__).parent / "postfix" / filename
+            ).read_bytes()
+        else:
+            _POSTFIX_FILES["README.md"] = (
+                Path(__file__).parent / "postfix" / filename
+            ).read_bytes()
+    elif filename.startswith("entrypoint.") or filename == "postfix-manpages.md":
+        if os_version != OsVersion.TUMBLEWEED and filename == "entrypoint.sles.sh":
+            _POSTFIX_FILES["entrypoint.sh"] = (
+                Path(__file__).parent / "postfix" / "entrypoint" / filename
+            ).read_bytes()
+        else:
+            _POSTFIX_FILES[filename] = (
+                Path(__file__).parent / "postfix" / "entrypoint" / filename
+            ).read_bytes()
+    else:
+        _POSTFIX_FILES[filename] = (
+            Path(__file__).parent / "postfix" / "entrypoint" / "ldap" / filename
+        ).read_bytes()
+
+
+POSTFIX_CONTAINERS = [
+    ApplicationStackContainer(
+        name="postfix",
+        package_name="postfix-image",
+        pretty_name="Postfix",
+        custom_description="Postfix container is fast and secure mail server, {based_on_container}.",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%postfix_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%postfix_version%%",
+                package_name="postfix",
+                parse_version="minor",
+            )
+        ],
+        package_list=[
+            name
+            for name in (
+                "ca-certificates",
+                "ca-certificates-mozilla",
+                "cyrus-sasl",
+                "cyrus-sasl-plain",
+                "ed",
+                "netcfg",
+                "postfix",
+                "postfix-ldap",
+                "postfix-lmdb",
+                "timezone",
+            )
+            + (
+                (
+                    "mandoc",
+                    "spamassassin-spamc",
+                    "spamass-milter",
+                )
+                if os_version == OsVersion.TUMBLEWEED
+                else ()
+            )
+        ],
+        entrypoint=["/entrypoint/entrypoint.sh && cat /entrypoint/postfix-manpages.md"],
+        cmd=["postfix", "start"],
+        extra_files=_POSTFIX_FILES,
+        support_level=SupportLevel.L3,  # TODO: confirm support level
```

Comment:
@dcermak, PR is ready for another round of review.

Summary of recent changes - https://github.com/SUSE/BCI-dockerfile-generator/pull/1046#issuecomment-2077268871

Ques - what is the support level for this container image?